apps/portals/elportal/src/App.scss

```diff
@@ -5,3 +5,6 @@
 
 // style_overrides should be last to ensure its styles have highest priority
 @use './config/style/style_overrides';
+
+// Merriweather font is used only on the ELITE Portal
```

Comment:
Could you also make a note in HomePageThemeProvider that this import is required (for the future engineer who applies this theme to other portal home pages)?

Reply:
https://github.com/Sage-Bionetworks/synapse-web-monorepo/pull/1450